### ISSUE TITLE
TurtleSim Pose is not displayed

### ISSUE THREAD

**Issue by ProfJust** [alexspirou/ROS_Melodic_Qt_GUI_Template#2]
The pose of the turtle1 is not displayed, the LCD stays x=0, y=0, theta=0.
With `$ rosnode info /ROS_QT_GUI_Template`  one can see that the `/turtle1/pose` - topic is not subscribed.
Adding mainwindow.cpp - line 20 after ros_f = new _Ros();
` ros_f->ultrasonic_subscriber(); `
will lead to a subscription but the turtlesim_pose_callback() is still not called.

What is missing?

**Comment by alexspirou** [comment]
Yeah, I should install a linux virtual machine to solve the issue.
